### DISCUSSION TITLE
HLL类型怎么查看原始值

### DISCUSSION THREAD

**Discussion by lsk984411392** [apache/doris#12472] [A - General / Q&A] [answered]
Hll类型的字段怎么查看原始值（类似bitmap_to_string方法可以查看bitmap的原始值）

**Comment by LemonLiTree** [comment, marked as answer]
目前不支持。通过hll_hash转化成hll，会因为hash算法不可逆，没法查看原始值。bitmap_to_string方法查看bitmap的原始值 也只是支持通过to_bitmap将int转化的bitmap可以查看，通过bitmap_hash转换的bitmap也是不支持查看原始值的

**Reply by lsk984411392** [reply]
好的 谢谢
